src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -144,6 +150,13 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
                 soqlQuery = 'SELECT npe03__Amount__c, npe03__Open_Ended_Status__c FROM npe03__Recurring_Donation__c WHERE Id > \'\'{0}\'\' ORDER BY Id LIMIT {1}';
             }
 
+            when DATA_IMPORTS {
```

Comment:
We should probably rename this to "GE_DATA_IMPORTS_BATCH" or something to be more descriptive - this telemetry isn't counting all DataImport__c records, but specifically counting DataImport__c records that are connected to a batch with a new-GE version. It's possible we will bring additional parameters (SGE_DATA_IMPORTS, etc) into this telemetry in the future

Reply:
I struggled with naming it actually, I think BGE_DATA_IMPORTS works.